src/middleware.ts

```
@@ -14,6 +15,20 @@ const intlMiddleware = createMiddleware({
 });
 
+
+const aj = arcjet({
+  // Get your site key from https://app.arcjet.com
+  // and set it as an environment variable rather than hard coding.
+  // See: https://nextjs.org/docs/app/building-your-application/configuring/environment-variables
+  key: process.env.ARCJET_KEY!,
+  rules: [
+    // Protect against common attacks with Arcjet Shield
+    shield({
+      mode: "LIVE", // will block requests. Use "DRY_RUN" to log only
```

ESLint

[error] Strings must use singlequote.

(style/quotes)